frontend/src/lib/components/dialogs/TraineeSelectModal.svelte

```diff
@@ -22,18 +21,16 @@
 	let selectedTrainees = $state<string[]>(
 		selectedParticipants.map((participant) => participant.uuid) ?? []
 	);
-	let isAddTraineeModalOpen = $state<boolean>(false);
-	let searchValue = writable(''); //TODO: dont use store
 
-	// Filter trainees based on the search value
-	const filteredTrainees = derived(searchValue, ($searchValue) => {
-		return $searchValue
-			? allTrainees.filter((trainee) =>
-					`${trainee.name} ${trainee.surname} ${trainee.username}`
-						.toLowerCase()
-						.includes($searchValue.toLowerCase())
-				)
-			: allTrainees;
+	let isAddTraineeModalOpen = $state(false);
+	let searchValue = $state('');
+
+	const filteredTrainees = $derived(() => {
```

Comment:
@hajir3 @michaljohnson This is a not how $derived is supposed to be used.

Please wait for proper review before merging FE PRs :(